eosclubhouse/libquest.py

```diff
@@ -1458,6 +1460,9 @@ class Quest(_Quest):
 
     '''
 
+    __app_repository__ = None
```

Comment:
Nit: The default value could be `config.DEFAULT_INSTALL_REPO`, which is currently `flathub`.

Reply:
Actually I think this change is needed. Because `None` is not a valid value for the repository, we are not handling if `None` anywhere.

Reply:
Yes, done.